CMakeLists.txt

```diff
@@ -38,13 +38,14 @@ endif()
 set(VERSION_TAG "" CACHE STRING "Build date for nightly versioning")
 set(EXTRA_DATA_DIR "" CACHE STRING "Directory for externalized data")
 set(INSTALL_LIB_DIR lib CACHE PATH "Installation directory for libraries")
-set(LOCAL_SDL_LIB "" CACHE STRING "Use local SDL library from this directory")
-set(LIBSTRACCIATELLA_TARGET "" CACHE STRING "Rust target architecture for libstracciatella")
-option(LOCAL_FLTK_LIB "Build with fltk lib" OFF)
-option(LOCAL_BOOST_LIB "Build with local boost lib" OFF)
+set(LOCAL_SDL_LIB "dependencies/lib-SDL2-2.0.8-VC" CACHE STRING "Use local SDL library from this directory")
+set(LIBSTRACCIATELLA_TARGET "x86_64-pc-windows-msvc" CACHE STRING "Rust target architecture for libstracciatella")
+
+option(LOCAL_FLTK_LIB "Build with fltk lib" ON)
+option(LOCAL_BOOST_LIB "Build with local boost lib" ON)
 option(LOCAL_RAPIDJSON_LIB "Build with local rapidjson lib" ON)
 option(LOCAL_GTEST_LIB "Build with local gtest lib" ON)
-option(WITH_UNITTESTS "Build with unittests" ON)
+option(WITH_UNITTESTS "Build with unittests" OFF)
```

Comment:
All but the first change look like mistakes. Options are cmake configure-time switches — why change the defaults?

Reply:
Yes sorry, I accidently commited the chanes I did to the build configuration, to make the build work with VS 2017, which supports cmake nativly. I'll revert them.